components/reusable-components/perk-card/PerkCard.tsx

```diff
@@ -3,7 +3,7 @@ import style from './PerkCard.module.scss';
 
 interface Props {
   perkCardText: string;
-  perkCardIcon: any;
+  perkCardIcon: string;
```

Comment:
so we should have the same interface here and in components/module-components/Perk-Cards/PerkCards.tsx
you would just export it

Reply:
Interfaces should have more descriptive names. Instead use `PerkCardProps`.